ui/src/themes/index.js

```diff
@@ -20,6 +21,7 @@ export default {
   GreenTheme,
   GruvboxDarkTheme,
   LigeraTheme,
+	CatppuccinMacchiatoTheme,
```

Comment:
Besides fixing the formatting, please add the name in alphabetical order, as noted in the comment above. So, it should be put before `ElectricPurpleTheme`